templates/page_header/page-header.tpl.php

```diff
@@ -8,6 +8,29 @@
 <div class="ecl-page-header">
   <?php print render($breadcrumb); ?>
   <div class="ecl-page-header__body">
-      <?php print render($page_header); ?>
-    </div>
+      <?php if (!empty($identity)): ?>
+        <div class="ecl-page-header__identity">
+          <?php print render($identity); ?>
+        </div>
+      <?php endif; ?>
+      <?php if (!empty($meta)): ?>
+        <div class="ecl-page-header__meta">
+          <?php print render($meta); ?>
+        </div>
+      <?php endif; ?>
+      <?php if (!empty($title)): ?>
+        <div class="ecl-page-header__title">
+          <?php print render($title_prefix); ?>
```

Comment:
prefix, suffix what do we foreseen here? The ecl component does not allow that; at least inside the `<div class="ecl-page-header__title">`.
If we do not have a clear use case, we should remove them.